bricksrc/parameter.py

```diff
@@ -77,11 +90,250 @@
                             TAG.Min,
                             TAG.Load,
                             TAG.Parameter,
-                            TAG.Setpoint,
                         ],
                     },
                 },
             },
+            "Deadband_Parameter": {
```

Comment:
I think we don't have to feel obligated to add "Parameter" in names. Just `Deadband` sounds good to me and, that way, it almost feels like an upgrade from the previous name instead of changing.